packages/concerto-cto/lib/parser.pegjs

```diff
@@ -1687,7 +1688,10 @@ FromUri
   = FromToken __ u:$URI __ {
     return u;
   }
+  
 
+  
+    
```

Comment:
Please clean up the extra whitespace

Reply:
Sure. 
Are we using any package that does this automatically before committing the code, something like a prettier package?

Reply:
Not for pegjs files unfortunately. Our linter only works with .js AFAIK